### PR TITLE
build doc enhancements

### PR DESCRIPTION
**Pull request by fayeshine**
build doc enhancements, described in #2083

### CODE REVIEW

File: tensorflow/g3doc/get_started/os_setup.md [hunk 1]
```diff
@@ -435,7 +435,10 @@ binary path.
 #### Install other dependencies
 
 ```bash
-$ sudo apt-get install python-numpy swig python-dev
+For Python 2.7:
```
Comment (martinwicke) [inline tensorflow/g3doc/get_started/os_setup.md]: Can you add a # in front of the lines that are not commands on the command line?

Comment (fayeshine) [inline tensorflow/g3doc/get_started/os_setup.md]: @martinwicke Sure, done.

**Comment by caisq** [comment]
@tensorflow-jenkins test this please

**Comment by fayeshine** [comment]
@martinwicke OK to merge?

**Comment by vrv** [comment]
there are conflicts :(

**Pull request closed without merging**